packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -2039,7 +2092,14 @@ void main() {
       // is not yet in the tree.
       await tester.pump();
 
-      final SemanticsNode semantics = tester.getSemantics(find.byType(UiKitView));
+      final SemanticsNode semantics = tester.getSemantics(
+        find.descendant(
+          of: find.byType(UiKitView),
+          matching: find.byWidgetPredicate(
+              (Widget widget) => widget.runtimeType.toString() == '_UiKitPlatformView',
+          ),
+        ),
```

Comment:
Why is this change needed? Does adding a focus node changing how semantics nodes are created?

Reply:
Yes, when we wrap it under a `Focus` node, the original `_UiKitPlatformView` got nested one level deeper in the tree

Reply:
oh I see `find.byType` is not recursive.